faculty_cli/cli.py

```diff
@@ -735,6 +735,13 @@ def ssh(project, server):
             tablefmt="plain",
         )
     )
+    click.echo(
+        "\n"
+        + "Login to the server with:\n"
+        + "ssh {}@{} -p {} -o UserKnownHostsFile={} -o StrictHostKeyChecking=no".format(
+            details.username, details.hostname, details.port, os.devnull
+        )
+    )
```

Comment:
Might this be a little tidier as:

```suggestion
    click.echo("")
    click.echo("Login to the server with:")
    click.echo(
        "ssh {}@{} -p {} -o UserKnownHostsFile={} -o StrictHostKeyChecking=no".format(
            details.username, details.hostname, details.port, os.devnull
        )
    )
```

We already compose the output like this in https://github.com/facultyai/faculty-cli/blob/master/faculty_cli/cli.py#L851

Reply:
Ah, thanks for the example! I was wondering of a setup like this, but haven't looked deep enough. Makes complete sense!